myRental/test/test.js

```diff
@@ -88,58 +84,27 @@ describe('requestController', () => {
             });
         });
     });
-});
-
-function generateFakeUserRequest() {
-    let fakePass = faker.internet.password();
-    return {
-        'body': {
-            'password': encrypt(fakePass),
-            'email': faker.internet.email(),
-            'username': faker.internet.userName(),
-            'address': faker.address.streetAddress(),
-            'state': faker.address.state(),
-            'zip': faker.address.zipCode(),
-            'city': faker.address.city(),
-        },
-        'session': {
-            'loggedIn': false,
-        },
-        'realPass': fakePass,
-    };
-}
-
-function generateFakeServiceRequest() {
-    return {
-        'body': {
-            'serviceRequest': {
-                'lawnCare': {
-                    'height': faker.random.number(),
-                    'pattern': "stripe",
-                    'fertilize': false,
-                    'water': false,
-                    'seeds': false,
-                    'removeWeeds': false,
-                    'misc': "",
-                }
-            },
-            'address': faker.address.streetAddress(),
-            'state': faker.address.state(),
-            'zip': faker.address.zipCode(),
-            'city': faker.address.city(),
-            'price': faker.random.number(),
-        },
-        'session': {
-            'loggedIn': true,
-            'userId': 1,
-        },
-    }
-}
 
-function encrypt(password) {
-    let cipher = crypto.createCipher(config.client_side_encryption.algorithm,
-            config.client_side_encryption.password);
-    let crypted = cipher.update(password, 'utf-8', 'hex');
-        crypted += cipher.final('hex');
-        return crypted;
-}
+    describe('post -> /searchRequest', () => {
+        it('A set of requests should be created and be searched', (done) => {
+            let testUserRequest = helperFunctions.generateFakeUserRequest();
+            let serviceRequest1 = helperFunctions.generateFakeServiceRequest1();
+            let serviceRequest2 = helperFunctions.generateFakeServiceRequest2();
+            let serviceRequest3 = helperFunctions.generateFakeServiceRequest3();
+            database.registerUser(testUserRequest, (cb) => {
+                database.handleRequest(serviceRequest1, (cb) => {
+                    database.handleRequest(serviceRequest2, (cb) => {
+                        database.handleRequest(serviceRequest3, (cb) => {
+                            database.searchForRequests(helperFunctions.generateSearchRequest(), (models) => {
+                                assert.equal(models.data[0].streetAddress, '201 E 17th St N');
```

Comment:
Is there a way to use a random address?

Reply:
I dont think making it random is practical.